browser/ui/tabs/BUILD.gn

```diff
@@ -42,7 +42,8 @@ source_set("tabs") {
 
 buildflag_header("buildflags") {
   header = "buildflags.h"
-  if (brave_channel == "development" || brave_channel == "nightly") {
+  if (brave_channel == "development" || brave_channel == "nightly" ||
+      brave_channel == "beta") {
```

Comment:
@simonhong Do you think it'd be better to remove this flag now, so that we don't need to fix this when we need it on the official channel too?

Reply:
Hmm, how about removing this after seeing what happens in beta? :)

Reply:
Yeah, that'd be better.